sklearn/utils/_testing.py

```diff
@@ -760,6 +776,10 @@ def assert_docstring_consistency(
         List of returns to be excluded. If None, no returns are excluded.
         Can only be set if `include_returns` is True.
 
+    description_regex : str, default=""
```

Comment:
This might be interested to mention something in the summary of the docstring as well.

Reply:
I'm not sure about the name here. I'm wondering if using the term "pattern" instead of "regex" would be more appropriate since this actually the part that we are passing in the function to feed a regular expression search.

Reply:
Since we already use shorted name maybe `descr_regex_pattern` would be what I would prefer.

Reply:
Done!